mods/cnc/weapons/other.yaml

```diff
@@ -112,19 +56,28 @@ Napalm:
 		Spread: 341
 		Damage: 30
 		Falloff: 1000, 368, 135, 50, 18, 7, 0
-		ValidTargets: Ground, Water, Trees
 		Versus:
-			None: 100
-			Wood: 100
-			Light: 100
 			Heavy: 80
-		DamageTypes: Prone50Percent, TriggerProne, FireDeath, Incendiary
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: Scorch
-		InvalidTargets: Vehicle, Structure, Wall
 	Warhead@3Eff: CreateEffect
 		Explosions: med_napalm
-		ImpactSounds: flamer2.aud
+
+Chemspray:
+	Inherits: ^FlameWeapon
```

Comment:
This adds a new `InvalidTargets: Wall` to the weapon and removes `Husk, Trees, Creep` from the smudge `InvalidTargets`.